apps/frontend/src/app/5_pages/RewardsPage/hooks/useGetSovGain.ts

```diff
@@ -0,0 +1,32 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { formatUnits } from 'ethers/lib/utils';
+
+import { useAccount } from '../../../../hooks/useAccount';
+import { useLoadContract } from '../../../../hooks/useLoadContract';
+import { getRskChainId } from '../../../../utils/chain';
+
+export const useGetSovGain = () => {
+  const { account } = useAccount();
+  const [sovGain, setSOVGain] = useState('0');
+
+  const stabilityPool = useLoadContract(
+    'stabilityPool',
+    'zero',
+    getRskChainId(),
+  );
```

Comment:
```suggestion
  const stabilityPool = useLoadContract(
    'stabilityPool',
    'zero'
  );
```
No need to specify the chain id (unless you want to use something else than RSK).